### PR TITLE
Separate traceback display from debug output level

### PR DESCRIPTION
**Pull request by stnatic**
References issue #1200 

This pull request doesn't update the documentation. Could you let me know in the comments if this solution is OK and then I would adjust documentation?

### CODE REVIEW

File: fabric/utils.py [hunk 1]
```diff
@@ -325,9 +325,11 @@ def error(message, func=None, exception=None, stdout=None, stderr=None):
     import fabric.state
     if func is None:
         func = fabric.state.env.warn_only and warn or abort
-    # If debug printing is on, append a traceback to the message
-    if fabric.state.output.debug:
-        message += "\n\n" + format_exc()
+    # If exception printing is on, append a traceback to the message
+    if fabric.state.output.exceptions:
```
Comment (bitprophet) [inline fabric/utils.py]: Let's preserve backwards compatibility by making this line say `if fabric.state.output.exceptions or fabric.state.output.debug` :)

**Comment by bitprophet** [comment]
This seems reasonable to me, so yea, please update the docs & make a changelog entry. Thanks!

**Comment by stnatic** [comment]
@bitprophet, thanks for feedback! I've added requested changes

**Comment by bitprophet** [comment]
Sweet deal, thanks!

**Comment by bitprophet** [comment]
I just updated the docs to adhere more cleanly with existing debug info, afb293f009395775390f4a811f5571fe2dcf1cad - FYI.

**Pull request merged**